docs/aws-config.md

```diff
@@ -139,9 +139,9 @@ kubectl create secret generic cloud-credentials \
 
 Specify the following values in the example files:
 
-* In `examples/aws/00-ark-config.yaml`:
+* In `examples/aws/00-ark-config.yaml` and `examples/aws/00-ark-backupstoragelocation.yaml`:
```

Comment:
I think the idea with the number prefix on each YAML file was to have a unique one per file (00, 10, 20, 30, ...) although TBH I can't even remember why we did this - presumably to specify an order - @ncdc you remember the details?

Reply:
Order makes sense; would 01 be better for this than 00?

Reply:
yeah I think that'd be fine.  Or you could go `05` if you want to leave some room on either side in case something else comes up that needs to go in between (unlikely, but who knows)